service/docs/source/geopm_pio.7.rst

```diff
@@ -177,6 +177,9 @@ Descriptions Of High Level Aliases
 ``GPU_CORE_FREQENCY_MIN_AVAIL``
```

Comment:
```suggestion
``GPU_CORE_FREQUENCY_MIN_AVAIL``
```
MAX_AVAIL is also misspelled.  These all need to be fixed in platform IO.
Thankfully it looks like this misspelling is only in code related to documentation.  The right string is passed to the alias registration methods.